storage/api.ts

```diff
@@ -1,11 +1,13 @@
 import {CloudinaryResponse, Quality} from "./types";
 
 export default {
-  upload: (file: File, quality: Quality) => {
+  upload: (tenantSlug: String, file: File, quality: Quality) => {
```

Comment:
aca le podemos poner solo tenant, el type en realidad no es `String` sino `string` igualmente lo mejor en este caso es importar el type `Tenant` y darle como type `Tenant['slug']`
```suggestion
  upload: (tenantSlug: Tenant['slug'], file: File, quality: Quality) => {
```